### DISCUSSION TITLE
Android或iOS版SDK中App schema配置

### DISCUSSION THREAD

**Discussion by kingsman2019** [white-shiro-bai/ghost_sa#31] [Q&A] [answered]
请教下哈，如果用Android或ios版SDK的时候，官方文档提到要在App的配置文件里配一个项目schema值, 正常是从部署的神策后台管理端生成的一个随机串；那如果是用鬼策后台的话，这个schema值怎么产生？
![image](https://user-images.githubusercontent.com/67091997/126119745-311878ed-9840-4fe7-84eb-bbc0434d7852.png)

**Comment by white-shiro-bai** [comment, marked as answer]
该功能官方介绍主要用于调试模式，可视化全埋点等功能。
https://manual.sensorsdata.cn/sa/latest/tech_check_debug_set-22253693.html
因为鬼策不支持这两个功能，所以scheme值随便给个文本就行。


鬼策用户，前端埋点时如果需要直接测试是否收到以及数据样式可以参考
https://github.com/white-shiro-bai/ghost_sa/blob/master/docs/%E4%BB%8E%E9%9B%B6%E5%BC%80%E5%A7%8B%E7%9A%84%E6%95%B0%E6%8D%AE%E5%88%86%E6%9E%90/%E9%AC%BC%E7%AD%96%E6%A0%B7%E6%9D%BF%E5%9F%8B%E7%82%B9%E8%A1%A8.xlsx
中埋点初始化，埋点校验部分的内容。

**Comment by white-shiro-bai** [comment]
![image](https://user-images.githubusercontent.com/40188518/126123544-515f7532-7cc2-480d-b8d1-f7ea610b14af.png)

**Reply by kingsman2019** [reply]
收到，解决了，多谢
